bittensor/_neuron/text/core_validator/__init__.py

```diff
@@ -739,21 +770,25 @@ def weights_table(self, sample_uids, sample_weights, include_uids=None, num_rows
         # === Weight table ===
         # Prints exponential moving average statistics of valid neurons and latest weights
         _neuron_stats = {}
+        uid_weights = []  # (uid, weight) tuples for sorting to find top/bottom weights
         unvalidated = []
         for uid, weight in zip(sample_uids.tolist(), sample_weights.tolist()):
             if uid in self.neuron_stats:
                 _neuron_stats[uid] = {k: v for k, v in self.neuron_stats[uid].items()}
                 _neuron_stats[uid]['weight'] = weight
+                uid_weights += [(uid, weight)]
             else:
                 unvalidated += [uid]
 
-        avail_include_uids = None
         if include_uids is not None and num_rows is not None:
-            avail_include_uids = list(set(_neuron_stats.keys()) & set(include_uids))  # exclude include_uids with no stats
+            sorted_uids = sorted(uid_weights, key=lambda tup: tup[1])
+            top_bottom_uids = [_uid for _uid, _ in sorted_uids[:5] + sorted_uids[-10:]]
```

Comment:
Is there a reason we want to include the bottom uids in the weights table?

Reply:
Viewing the bottom 5 weights in the validator debug weight table is useful to confirm the exclusion quantile and associated Shapley values, losses, base parameters, and synergy values, which miners can use to choose models for newly registered server.